aws/sqldatabases.bicep

```diff
@@ -50,6 +50,9 @@ param allocatedStorage string = '20'
 @description('Database license model')
 param licenseModel string = 'license-included'
 
+@description('The number of days for which automated backups are retained. Setting this parameter to a positive number enables backups. Defaults to 1.')
```

Comment:
Could you include this in the suggestion as well https://github.com/radius-project/recipes/pull/50#discussion_r1399735395?

Reply:
I checked the spec for this resource and it doesn't look like there's a maximum - I think the link yetkin linked is the C# SDK

```json
    "BackupRetentionPeriod": {
      "type": "integer",
      "minimum": 0,
      "default": 1,
      "description": "The number of days for which automated backups are retained. Setting this parameter to a positive number enables backups. Setting this parameter to 0 disables automated backups."
    },
```

https://us-west-2.console.aws.amazon.com/cloudformation/home?region=us-west-2#/registry/activated-extensions/details/schema?arn=arn%3Aaws%3Acloudformation%3Aus-west-2%3A%3Atype%2Fresource%2FAWS-RDS-DBInstance

Reply:
https://docs.aws.amazon.com/cli/latest/reference/rds/create-db-instance.html - this describes max as 35. 

```
Constraints:

Must be a value from 0 to 35.
Can't be set to 0 if the DB instance is a source to read replicas.
Can't be set to 0 for an RDS Custom for Oracle DB instance.
```